### ISSUE TITLE
Graph points color issue

### ISSUE THREAD

**Issue by wiuwiu** [kalkih/mini-graph-card#38]
There is an issue with graph points color in case of use `line_color_above`. Graph points should have the color same as graph but they have same as `line_color`.

![image](https://user-images.githubusercontent.com/21005444/52352123-52f18b80-2a2c-11e9-99b8-10ad82c7911a.png)

**Comment by kalkih** [comment]
But that's a feature!
Jokes aside 😉, could you also post the card configuration ?
Thanks!

**Comment by wiuwiu** [comment]
@kalkih this card is AWESOME and thank you for you work.
As for the data graph pints they didn't change when the graph line changes.
Here is my config:
```
              - type: custom:mini-graph-card
                entity: sensor.living_room_devices_power
                hours_to_show: 12
                points_per_hour: 2
                hour24: true
                animate: true
                show:
                  icon: false
                  name: false
                line_color: '#2ca529'
                line_color_above:
                  - value: 50
                    color: '#ff9800'
                  - value: 100
                    color: '#e74c3c'
```

**Comment by kalkih** [comment]
Should be fixed in [v0.2.2](https://github.com/kalkih/mini-graph-card/releases/tag/v0.2.2)
